doc/classes/TextureRect.xml

```diff
@@ -35,17 +35,17 @@
 		<constant name="EXPAND_IGNORE_SIZE" value="1" enum="ExpandMode">
 			The size of the texture won't be considered for minimum size calculation, so the [TextureRect] can be shrunk down past the texture size.
 		</constant>
-		<constant name="EXPAND_FIT_WIDTH" value="2" enum="ExpandMode">
+		<constant name="EXPAND_FIT_HEIGHT" value="2" enum="ExpandMode">
```

Comment:
Values and order should match `enum ExpandMode` order. You should swap description texts, not names.